src/viewport.cpp

```diff
@@ -1199,6 +1218,7 @@ void PixelFinder::CollectVoxel(const Voxel *voxel, const XYZPoint16 &voxel_pos,
 			dd.base.x = this->rect.base.x - xnorth - x_off;
 			dd.base.y = this->rect.base.y - ynorth - y_off;
 			dd.recolour = nullptr;
+			dd.highlight = false;
```

Comment:
perhaps it would be better to make dd.highlight = false the default? (set it in the constructor)